netatmo.js

```diff
@@ -327,9 +329,9 @@ netatmo.prototype.getMeasure = function(options, callback) {
     form: form,
   }, function(err, response, body) {
     if (err || response.statusCode != 200) {
-      console.log(body);
-      this.emit("error", new Error("getMeasure error: " + response.statusCode));
-      return this;
+      var error = this.handleRequestError(err,response,body,"getMeasure error");
+      if (callback) callback(error);
```

Comment:
Sure, will do if you think this fits your style better.
I personally find it a bit too verbose and prefer this syntax.